bigquery/bigquery_create_table/main.tf

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START bigquery_create_table]
+resource "google_bigquery_dataset" "default" {
+  dataset_id                      = "mydataset"
+  default_partition_expiration_ms = 2592000000  # 30 days
+  default_table_expiration_ms     = 31536000000 # 365 days
+  description                     = "dataset description"
+  location                        = "US"
+  max_time_travel_hours           = 96 # 4 days
+
+  labels = {
+    billing_group = "accounting",
+    pii           = "sensitive"
+  }
+}
+
+resource "google_bigquery_table" "default" {
```

Comment:
Please add:

`deletion_protection=false`

with a comment to set it to `true` in production.

Here is the logged error in the test run:

```
Step #5 - "test-samples-3": Error: cannot destroy instance without setting deletion_protection=false and running `terraform apply`
Step #5 - "test-samples-3": }
Step #5 - "test-samples-3":     destroy.go:11: 
Step #5 - "test-samples-3":         	Error Trace:	/workspace/test/integration/destroy.go:11
Step #5 - "test-samples-3":         	            				/workspace/test/integration/terraform.go:350
Step #5 - "test-samples-3":         	            				/workspace/test/integration/terraform.go:413
Step #5 - "test-samples-3":         	            				/workspace/test/integration/terraform.go:426
Step #5 - "test-samples-3":         	            				/workspace/test/integration/stages.go:31
Step #5 - "test-samples-3":         	            				/workspace/test/integration/terraform.go:429
Step #5 - "test-samples-3":         	Error:      	Received unexpected error:
Step #5 - "test-samples-3":         	            	FatalError{Underlying: error while running command: exit status 1; 
Step #5 - "test-samples-3":         	            	Error: cannot destroy instance without setting deletion_protection=false and running `terraform apply`
Step #5 - "test-samples-3":         	            	}
Step #5 - "test-samples-3":         	Test:       	TestSamples/3/bigquery_create_table
```